### ISSUE TITLE
[CM-MLPerf] Reproduce all reference implementations of the MLPerf inference benchmark using CM

### ISSUE THREAD

**Issue by gfursin** [mlcommons/ck#366]
As discussed during the last inference WG meeting, we continue adding CM components and workflows for all reference implementation of the MLPerf inference benchmark. 

We then gradually test all reference implementations across different hardware, OS, compilers, frameworks and libraries.

We will be adding new reproducibility reports [here](https://github.com/mlcommons/ck/blob/master/docs/mlperf-automation-cm/reproducibility.md#original-python-front-end-for-loadgen)

CC @pgmpablo157321

**Comment by arjunsuresh** [comment]
We have so far tested
* ResNet50 onnxruntime and tensorflow
* Retinanet onnxruntime and pytorch
* Bert onnxruntime

**Comment by gfursin** [comment]
Added to our new roadmap: https://github.com/mlcommons/ck/issues/536

**Comment by arjunsuresh** [comment]
We have now covered
1. ResNet50, [onnxruntime, tensorflow], [cuda,cpu]
2. RetinaNet, [onnxruntime, pytorch], [cuda,cpu]
3. Bert, [onnxruntime, tensorflow, pytorch], [cuda,cpu]
4. 3d-unet, [onnxruntime,pytorch,tensorflow], [cuda,cpu]
5. rnnt, [pytorch], [cpu] 
6. dlrm, [pytorch], [cpu]

**Comment by gfursin** [comment]
@arjunsuresh . I believe we have automated all ref implementations?

**Comment by arjunsuresh** [comment]
yes @gfursin . We have GPT-J, GPT-3 and DLRMv2 in the pipeline though.
